lib/req_athena.ex

```diff
@@ -183,8 +183,10 @@ defmodule ReqAthena do
         [parameters, :os.system_time()]
       end
 
+    # We need a minimum of 32 chars for the token, so we use MD5.
     parameters
-    |> :erlang.term_to_binary()
+    |> :erlang.phash2()
+    |> Integer.to_string()
```

Comment:
I am not sure if this change is safe. Phash2 is not unique and it will reduce the amount of bytes information given to md5.

Reply:
Oh, I see the problem. [This token](https://docs.aws.amazon.com/athena/latest/APIReference/API_StartQueryExecution.html#API_StartQueryExecution_RequestParameters) is going to be use to ensure that the query is idempotent, so I believe the commit I made - e695f20812ff6ae210677f06c1f16f900d85c74f is phashing each key and value - is going to minimize the problem. WDYT?

Reply:
phash2 does not guarantee uniqueness and the point of this hash is to be unique, so I would generally prefer to avoid it. What is the issue with the current code?

Reply:
The issue is that it is breaking across different OTP versions - probably due the difference in maps ordering. I'm not sure how to fix that. Maybe recursive sorting the maps?

Reply:
I would change the tests to assert on the values possible across OTP versions. So we check for A or B. Ultimately it is a test issue, not a code one. :)

Reply:
I agree. I rolled back the changes related to that, and I should update later this week :)